libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -253,6 +258,33 @@ describe('DropdownComponent', () => {
       });
     });
 
+    describe('when configured with cardAligment', () => {
+      it('should render card with left alignment', () => {
+        spectator.component.cardAlign = cardAlignment.left;
+        spectator.component['state'] = openState.open;
+        spectator.detectChanges();
+        const buttonRect = buttonElement.getBoundingClientRect();
+        const card = spectator.query('kirby-card');
+        const cardRect = card.getBoundingClientRect();
+        expect(cardRect.left).toEqual(buttonRect.left);
+      });
+
+      it('should render card with right alignment', (done) => {
+        spectator.setInput('cardAlign', cardAlignment.right);
+        spectator.element.style.cssFloat = 'right';
+        spectator.component.open();
+        spectator.detectChanges();
+        setTimeout(() => {
```

Comment:
Why not just setting the state to open as in the other test? 🤔

Reply:
Also a copy/paste.. made it consistent